addons/full-signature/working.js

```diff
@@ -0,0 +1,32 @@
+export default async function ({ addon, global, console }) {
+  document.querySelector(".activity-stream").insertAdjacentHTML(
```

Comment:
Please use appendChild

Reply:
Nothing wrong with insertAdjacentHTML. It is also safe.